benchmark/e2e-ci/benchmark-run.js

```diff
@@ -126,19 +126,21 @@ async function main () {
   console.log('Workflows in progress:', workflowsInProgress)
   const { total_count: numWorkflows, workflow_runs: workflows } = workflowsInProgress
   if (numWorkflows === 0) {
-    throw new Error('Could not find the triggered job')
+    throw new Error('Could not find the triggered workflow')
   }
   // Pick the first one (most recently triggered one)
   const [{ id: runId } = {}] = workflows
 
-  console.log('Waiting for the workflow to finish.')
-  console.log(`Job URL: https://github.com/DataDog/test-environment/actions/runs/${runId}`)
-  // Poll every 15 seconds until we have a finished status
+  console.log(`Workflow URL: https://github.com/DataDog/test-environment/actions/runs/${runId}`)
+
+  // Wait an initial 1 minute, because we're sure it won't finish earlier
+  await wait(60000)
```

Comment:
I've decreased the polling interval to finish faster. Not to fry the API I've added this delay at the beginning